visualvm/launcher/visualvm

```diff
@@ -63,7 +63,11 @@ esac
 
 
 if [ -f "$progdir"/../lib/visualvm/etc/visualvm.conf ] ; then
-    visualvm_jdkhome="$basedir"
+    if [[ -d "$JAVA_HOME" && -f "$JAVA_HOME/bin/java" ]]; then
```

Comment:
The Windows version also takes JDK_HOME into consideration, maybe it would be better to have the same behaviour.